app/models/volunteer.rb

```diff
@@ -31,4 +35,8 @@ def self.group_accompaniment
   def self.human_boolean(boolean)
     boolean ? I18n.t('simple_form.yes') : I18n.t('simple_form.no')
   end
+
+  def to_s
```

Comment:
needed for confirmation message and many other things. this is standard practice...